assets/src/blocks/Timeline/TimelineEditor.js

```diff
@@ -0,0 +1,124 @@
+import {RawHTML, useCallback, useState} from '@wordpress/element';
+import {
+	PanelBody,
+	SelectControl,
+	CheckboxControl,
+	Tooltip
+} from '@wordpress/components';
+import { InspectorControls } from '@wordpress/block-editor';
+
+import { URLInput } from '../../components/URLInput/URLInput';
+import { Timeline } from './Timeline';
+import { languages } from './TimelineLanguages'
+import { URLDescriptionHelp } from './URLDescriptionHelp'
+import { debounce } from 'lodash';
+
+const { RichText } = wp.blockEditor;
+const { __ } = wp.i18n;
+
+const positions = [
+	{label: 'Bottom', value: 'bottom'},
+	{label: 'Top', value: 'top'},
+];
+
+const renderEdit = ({ google_sheets_url, language, timenav_position, start_at_end }, toAttribute) => {
+  // Using a state to prevent the input losing the cursor position, a React issue reported multiple times
+  const [ sheetURL, setSheetURL ] = useState(google_sheets_url);
+  const debounceSheetURLUpdate = useCallback(debounce(toAttribute('google_sheets_url'), 300), []);
+
+  return (
+    <InspectorControls>
+      <PanelBody title={__('Setting', 'planet4-blocks-backend')}>
+        <URLInput
+          label={__('Google Sheets URL', 'planet4-blocks-backend')}
+          placeholder={__('Enter URL', 'planet4-blocks-backend')}
+          help={ <RawHTML>{URLDescriptionHelp}</RawHTML> }
```

Comment:
React makes a `div` for `RawHTML` (which shouldn't be here according to [the doc](https://github.com/WordPress/gutenberg/blob/375eaa0d870282e12cd461a53463d3e10e776c41/packages/element/src/raw-html.js#L21) ?) and then complains that a `div` is in a `p` `Warning: validateDOMNesting(...): <div> cannot appear as a descendant of <p>.` :man_shrugging: